### DISCUSSION TITLE
Instinct MI100 server fan control

### DISCUSSION THREAD

**Discussion by ccbadd** [ROCm/ROCm#2384] [Q&A] [answered]
I have been trying to find out how to read the temperatures of an MI100 under Ubuntu 22.04 using LM Sensors.  I have the lm_sensors package installed but when I scan for temperatures is says none found when probing the card.  I want to use these temps to control the fans of an Asus ECS4000G3 server.  I'd also like to know the max operating temp of these cards but can't seem to find that info anywhere.  Thanks.

**Comment by zichguan-amd** [comment, marked as answer]
Hi @ccbadd, you can monitor your card's temperatures with `amd-smi metric -t` and the power and temperature limits with `amd-smi static -l`
